realm/realm-library/src/main/java/io/realm/RealmSchema.java

```diff
@@ -54,7 +54,7 @@
      *
      * @return the set of all classes in this Realm or no RealmObject classes can be saved in the Realm.
      */
-    public abstract Set<? extends RealmObjectSchema> getAll();
```

Comment:
The point is that we can't introduce any breaking changes without bumping major version.

Changing return type from `Set <RealmObjectSchema>` to `Set<? extends RealmObjectSchema>` is a breaking change. Users might have a code `Set<RealmObjectSchema> allObjectSchema = realm.getSchema().getAll();` and this line causes compile error in 3.1.